projects/burnr/src/components/ReceiveFundsForm.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useContext } from 'react';
+
+import { AccountContext } from '../utils/contexts';
+
+import QRCode from 'qrcode.react';
+// import { InputFunds } from '../components';
+// import { useBalance } from '../hooks'
+import { makeStyles, createStyles, Theme, Grid } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) =>
+	createStyles({
+		container: {
+			marginTop: theme.spacing(3),
+		},
+		formSubmitContainer: {
+			display: 'flex',
+			marginTop: theme.spacing(3),
+			alignContent: 'center',
+            justifyContent: 'center',
+		},
+        qr: {
+            color: theme.palette.getContrastText(theme.palette.secondary.main),
+            backgroundColor: theme.palette.getContrastText(theme.palette.secondary.dark)
+        },
+		button: {
+			color: theme.palette.getContrastText(theme.palette.secondary.main),
+			'&:hover': {
+				color: theme.palette.getContrastText(theme.palette.secondary.dark),
+			},
+		},
+	})
+);
+  
+
+const ReceiveFundsForm: React.FunctionComponent = () => {
+    const { account } = useContext(AccountContext);
+    // const balanceArr = useBalance(account.userAddress)
+	const classes = useStyles();
+	return (
+		<Grid
+			component='form'
+			container
+			direction='column'
+			className={classes.container}
+		>
+			{/*<Grid item>
+				<InputFunds
+                    hidePercentages
+					total={100}
+					currency={ballanceArr[3]}
+				/>
+            </Grid>*/}
```

Comment:
Did you mean to leave this commented out code in?  Looks like you have an actual implementation below so its confusing.  I prefer not to keep commented code in (without an explanation fir it)... you can always look back through the git history if you want to recover something that was there in the past.

Reply:
There are also unused styles. I don't think there's need to keep the commented lines.

```js
import React, { useContext } from 'react';

import { AccountContext } from '../utils/contexts';

import QRCode from 'qrcode.react';
import { makeStyles, createStyles, Theme, Grid } from '@material-ui/core';

const useStyles = makeStyles((theme: Theme) =>
	createStyles({
		qrBox: {
			alignContent: 'center',
			justifyContent: 'center',
			paddingTop: theme.spacing(3)
		},
	})
);
  

const ReceiveFundsForm: React.FunctionComponent = () => {
	const { account } = useContext(AccountContext);
	const classes = useStyles();
	return (
		<Grid
			container
			className={classes.qrBox}
		>
			<Grid item>
				<QRCode
					value={account.userAddress}
					size={400}
					bgColor={"#ffffff"}
					fgColor={"#000000"}
					includeMargin={true}
					renderAs={"svg"}
				/>
			</Grid>
		</Grid>
	);
};

export default ReceiveFundsForm;
```

Reply:
> Did you mean to leave this commented out code in? Looks like you have an actual implementation below so its confusing. I prefer not to keep commented code in (without an explanation fir it)... you can always look back through the git history if you want to recover something that was there in the past.

I was not sure if the input should stay there or not. I'll remove it and retrieve it from history if needed.